tesseract_collision/include/tesseract_collision/core/types.h

```diff
@@ -414,6 +304,8 @@ struct CollisionCheckConfig
   {
   }
 
+  /** @brief Identify how the collision margin data should be applied to the contact manager */
+  CollisionMarginOverrideType collision_margin_override_type{ CollisionMarginOverrideType::NONE };
```

Comment:
Do you think the default should be none or replace?

Reply:
This is probably fine the way it is

Reply:
Sounds good.